### DISCUSSION TITLE
blog/tweet-archive-in-r/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#131] [Comments]
# blog/tweet-archive-in-r/

Using R to read and visualize my Twitter archive data. Featuring {ggiraph}, {ggplot2}, {jsonlite}, {dplyr} and more&hellip;

https://www.garrickadenbuie.com/blog/tweet-archive-in-r/

**Comment by rmflight** [comment]
Bookmarks aren't included? I guess when bookmarks were added the data archive team didn't get to update the tool. That was actually the one piece of data I really wanted, as I'd started bookmarking more articles and how-to's instead of liking them as a bookmark holder. Crap.

**Reply by gadenbuie** [reply]
Yeah, can you believe that!? They're not easy to get from rtweet either 😒

**Reply by mpettis** [reply]
Not easy, but possible?  I'm using {rtweeet} to archive my Twitter account...

**Reply by gadenbuie** [reply]
Technically it's possible with the API in v2, but it's not implemented in rtweet yet. Follow this issue to learn more or hear about updates on that front: https://github.com/ropensci/rtweet/issues/344

@llrs says it's likely to be worked on next: https://twitter.com/lluis_revilla/status/1601145399309656064

**Reply by llrs** [reply]
Yes, I already started to work on it but now I will focus on finishing the next release and then work in this endpoint (and the other remaining bugs).

**Comment by mpettis** [comment]
This is great, thanks!  As a daily R user, I am not familiar with, nor can I find, `purrr::list_rbind()`.  You link to it here, but the page is not found.  Is this experimental, or new?  https://purrr.tidyverse.org/reference/list_c.html

**Reply by rmflight** [reply]
This is probably it:

> dev version of purrr that is slated to become version 1.0.0

**Reply by gadenbuie** [reply]
Yup! There are lots of cool new features in the upcoming purrr release, but the biggest change for me is going to be moving from `map_dfr()` to `map() |> list_rbind()`. I started the post intending to go all-in on the new syntax but old habits are hard to break.

https://twitter.com/hadleywickham/status/1590452780703285249

**Comment by Stramon1um** [comment]
Hi, super thanks for this guide!

I have an issue though, when I try to create the final *tidy_tweets*, the column related to created_at, and the column related to it (hour, day, etc) are empty, while in the in *tweet* I have that column the same format as you show here. Might it be a problem of strptime trying to get the time format? Thanks in advance!

**Reply by Stramon1um** [reply]
EDIT: solved! It was the datetime locale of my system. I set it up in English and now it was able to get the name of the day and month from tweet.js ;)
